### ISSUE TITLE
Deb file not building

### ISSUE THREAD

**Issue by muyiwao221** [trapexit/mergerfs#283]
I have no doubt that this is an issue on my system, however for some reason when I try to make the deb file, it doesn't complete and leaves me with:

dpkg-shlibdeps: error: no dependency information found for /lib/l
ibfuse.so.2 (used by debian/mergerfs/usr/bin/mergerfs)
dh_shlibdeps: dpkg-shlibdeps -Tdebian/mergerfs.substvars debian/m
ergerfs/usr/bin/mergerfs returned exit code 2ntu-trusty/mergerfs'
make[1]: *** [binary] Error 2
make[1]: Leaving directory `/root/mergerfs'
dpkg-buildpackage: error: debian/rules binary gave error exit sta
tus 2
make: *** [deb] Error 2

I really don't know quite what to make of this. From what I gather its an issue with my debian/rules file, but I also don't want to mess with anything. Hopefully you have seen this before because I don't really know where to go from here. Please let me know if there's a solution.

Thanks

**Comment by trapexit** [comment]
You need to provide more information. What distro, version? Do you have the build packages installed as described in the readme?

**Comment by muyiwao221** [comment]
Yeah sorry. Im running Ubuntu 14.0.4.3 - kernel 3.19.0-26.
I installed all the build packages using the command provided. In order, I ran:

`$ git clone https://github.com/trapexit/mergerfs.git`
`$ sudo apt-get install g++ pkg-config git git-buildpackage pandoc debhelper libfuse-dev libattr1-dev python
$ cd mergerfs
$ make deb`

And then after running make deb, I run into that issue. Let me know if you need anything else.

Thanks

**Comment by trapexit** [comment]
1) You don't need to build it yourself. I've packages for lots of platforms in the releases.
2) I've not seen this before. Has libfuse been modified? Do you have "libfuse2" package installed?

**Comment by muyiwao221** [comment]
alright I'll try using the package. Should have started with that in hindsight. And I think libfuse was modified now that I think about it. I was having issues with mhddfs and installed the mhddfs-nosegfault. However If i remember correctly that required libfuse2. Is that what is causing the issue?

Thanks

**Comment by trapexit** [comment]
I don't know. I'm not familiar with that error nor know what you've done to your system.

**Comment by muyiwao221** [comment]
Alright ill try to figure it out. Do I need libfuse2 or just libfuse?

**Comment by trapexit** [comment]
There is no 'libfuse' package. It's 'libfuse2'

**Comment by muyiwao221** [comment]
In that case then it shouldn't be modified. I have libfuse2 version 2.9.4.

**Comment by trapexit** [comment]
Clearly something is odd with your system because Ubuntu 14.04.3 is the primary development and build platform I use and it builds fine in every Debian system I've tried as you can see from the releases provided.

**Comment by muyiwao221** [comment]
Yeah I figured as much. I've never run into an issue like this while installing a package. Should I maybe try reinstalling libfuse? 

Even when i try to install the package, I get a fopen: Permission denied. - is that anything you've seen before?

I apologize to keep bugging you with questions, I understand you're busy too. I just really want to get this working. I do really appreciate your help.

Thanks

**Comment by trapexit** [comment]
What did you run to install it? "sudo dpkg -i <path_to_deb_file"?

**Comment by muyiwao221** [comment]
yea

**Comment by muyiwao221** [comment]
This is what I get:

`root@Apollo:~# sudo dpkg -i mergerfs_2.13.1.ubuntu-trusty_amd64.deb `
`(Reading database ... 188447 files and directories currently installed.)`
`Preparing to unpack mergerfs_2.13.1.ubuntu-trusty_amd64.deb ...s/usr/bin/mergerfs)`
`Unpacking mergerfs (2.13.1~ubuntu-trusty) over (2.13.1~ubuntu-trusty) ...xit code 2`
`Setting up mergerfs (2.13.1~ubuntu-trusty) ...`
`Processing triggers for man-db (2.6.7.1-1ubuntu1) ...`
`fopen: Permission denied`

**Comment by trapexit** [comment]
Sounds to me your system is in a bad / weird state. I don't know what could be causing that and I can't really comment further. BTW... you don't need to use "sudo" if you're already logged in as root.

**Comment by muyiwao221** [comment]
Yeah I know, I was just seeing if it made a difference in that instance. And alright, thanks for the help anyways. Ill troubleshoot further and see if I can find out what's going on. Thanks for the help though I really appreciate it.

**Comment by muyiwao221** [comment]
Ok. I got it installed. Fresh install and all. Now when I try to mount it, I get 
`fuse: mount failed: Invalid argument /media/hdd2`

I used the command:
```
mergerfs -o defaults /media/hdd1 /media/hdd2 /media/hdd3 /media/hdd4 /media/mootoo/
```
to see if there was an issue with my drives, though, I mounted mhddfs again and it worked. I know this is getting tedious so this is my last attempt. Any chance you've seen that one before?

Thanks
